test/Microsoft.Health.Fhir.R4.Tests.E2E/Microsoft.Health.Fhir.R4.Tests.E2E.csproj

```diff
@@ -34,7 +34,6 @@
     <PackageReference Include="System.Diagnostics.PerformanceCounter" Version="5.0.1" />
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\..\samples\apps\SmartLauncher\SmartLauncher.csproj" />
```

Comment:
>SmartLauncher [](http://example.com/codeflow?start=64&length=13)

!
There is two web project linked by E2E tests, *.Web and SmartLauncher.
Both projects has appsettings.json file which they put build directory.

During build it's fine, but during publishing nuget (new version coming with .net6) would raise error.

To prevent that error, I just remove SmartLauncher for been part of the tests. 

From what I see, we don't do any useful tests anyway.